apps/astarte_housekeeping/lib/astarte_housekeeping/queries.ex

```diff
@@ -836,10 +851,11 @@ defmodule Astarte.Housekeeping.Queries do
     # TODO: add support for creating the astarte keyspace with NetworkTopologyStrategy,
     # right now the replication factor is an integer so SimpleStrategy is always used
     astarte_keyspace_replication = Config.astarte_keyspace_replication_factor!()
+    instance_id = Config.astarte_instance_id!()
 
     with {:ok, replication_map_str} <- build_replication_map_str(astarte_keyspace_replication),
          query = """
-         CREATE KEYSPACE astarte
+         CREATE KEYSPACE #{instance_id}astarte
```

Comment:
The Astarte instance ID is arbitrary, so I suggest using ` CQLUtils.realm_name_to_keyspace_name("astarte", Config.astarte_instance_id!())` here too, just to make sure we do not go over the Scylla keyspace name length limit